docs/mdl-sass.md

```diff
@@ -0,0 +1,187 @@
+    gulp.src([
+      // Component handler
+      './app/styles/src/mdlComponentHandler.js',
+      // Base components
+      './app/styles/src/button/button.js',
+      './app/styles/src/checkbox/checkbox.js',
+      './app/styles/src/icon-toggle/icon-toggle.js',
+      './app/styles/src/menu/menu.js',
+      './app/styles/src/progress/progress.js',
+      './app/styles/src/radio/radio.js',
+      './app/styles/src/slider/slider.js',
+      './app/styles/src/spinner/spinner.js',
+      './app/styles/src/switch/switch.js',
+      './app/styles/src/tabs/tabs.js',
+      './app/styles/src/textfield/textfield.js',
+      './app/styles/src/tooltip/tooltip.js',
+      // Complex components (which reuse base components)
+      './app/styles/src/layout/layout.js',
+      './app/styles/src/data-table/data-table.js',
+      // And finally, the ripples
+      './app/styles/src/ripple/ripple.js',
+      // Other scripts,
+      './app/scripts/main.js'
+    ])
+```
+
+
+> Note: We are aware that it can feel a little suboptimal to reference the same set of source files twice in the above pipeline. We wil be looking at simplifying this workflow in a future release. 
+
+Similar to styles, you can comment out what you don't need here if you decide to only use a smaller set of components. 
+
+7. Finally, we can run `gulp serve` to preview our site or `gulp` to build a production version. Yay!
+
+## Bonus tips
+
+### Configuring colors
+
+You will probably want to configure the color theme used in MDL. 
+
+MDL supports Material Design's [color palette](https://www.google.com/design/spec/style/color.html#color-color-palette) through Sass variables. 
+
+If you find a color in the palette spec you would like to use e.g color name `Pink` with fill `500`, MDL exposes this as `$palette-purple-500`. Let's walk through customising the primary and accent colors for your theme.
+
+This can be done using your setup as follows:
+
+1. Run `gulp serve` to preview your site. 
+2. Open `app/styles/src/_variables.scss` in your Text Editor.
+3. Find `$color-primary`. It should be in the same block as two other color variables we can use for theming - `$color-primary-dark` and `$color-accent`.
+
+```
+$color-primary: $palette-indigo-500 !default;
+$color-primary-dark: $palette-indigo-700 !default;
+$color-accent: $palette-pink-A200 !default;
+```
+
+We can change out the default theme for a custom one by consulting `app/styles/src/_color-definitions.scss` for [Material Design color themes](https://www.google.com/design/spec/style/color.html) available. 
+
+For this example, we're going to change out `palette-indigo-500` for `palette-purple-500` and `palette-indigo-700` for `palette-deep-purple-700`.
+
+```
+$color-primary: $palette-purple-500 !default;
+$color-primary-dark: $palette-deep-purple-700 !default;
+$color-accent: $palette-pink-A200 !default;
+```
+
+Make the above change and hit save. The page should refresh, now showing you your customised theme in action. 
+
```

Comment:
You need to set two other colours: `$color-primary-contrast` and `$color-accent-contrast`. These are the colours for text that gets rendered on top of a solid block of primary or accent, respectively. You should set them to `$color-dark-contrast` if you've chosen a dark primary/accent, and `$color-light-contrast` if you've chosen a light primary/accent.